homepage/models.py

```diff
@@ -40,25 +44,47 @@ class Req_Code(models.IntegerChoices):
     req_code = models.SmallIntegerField(choices=Req_Code.choices, default=Req_Code.NONE)
 

```

Comment:
Missing ```__str__``` implementation for this class

Reply:
when you write it, don't miss creating an __str__ method that references the req_code as a word:
it was intended to be as written in the class Req_code only SIMU (0) is short for simultaneous.

Reply:
Yes, this is Deliberately.
In my opinion it is not needed.

if we want to add it, it can be done later.
if it only for the Django admin page - it is not necessary since we will add the "table view" for all models.

Reply:
@VeraTs I'm not sure I understand, I suggested adding the ```__str__``` method only to the Prerequisites class.
right now when I print a Prerequisites object through the consol it prints ```<Prerequisites: Prerequisites object (1)>```. @LeonRabinovich Anyway, an ```__str__``` method is nice to have not a must. Can be added for uniformity with the rest of the classes.

Reply:
it depicts an important relationship, and I think we'd prefer it to be legible in the shell

Reply:
@VeraTs 
I Added the __str__ implementation for the Prerequisites class.
However, i had some issues to implement this in the Req_Code class without making changes to the class itself - so i decided not to at this point.

If you think its useful,  i suggest that after this PR is merged - open a separate PR just for the Req_Code class.